index.js

```diff
@@ -961,6 +969,10 @@ function parseTestAsm (source, line) {
   return tests;
 }
 
+function removeEndNewline (str) {
+  return (str[str.length - 1] === '\n') ? str.slice(0, -1) : str;
```

Comment:
str.endsWith('\n') .. why only slice 1 newline?

Reply:
ok a7450b5aba478a050412f15c4ededaf87ee4c671 ... well we wouldn't want some buggy code to add a ton of newlines at the end of some output and the tests not picking up on that